src/v2/ion/i18nTransformer.js

```diff
@@ -152,6 +152,8 @@ const I18N_OVERRIDE_MAPPINGS = {
 
   'idx.email.verification.required': 'registration.complete.confirm.text',
   'security.access_denied': 'errors.E0000006',
+  // 'Service is in read only mode' => 'Server is unable to respond at the moment'.
+  'errors.E0000010': 'E0000010',
```

Comment:
Can you add the mock response where we get this i18n key back in the API to https://github.com/okta/okta-signin-widget/tree/master/playground/mocks/data/idp/idx

Reply:
It is a bit confusing that we have 2 versions of this message in the login bundle one with the prefix and one without 

```
errors.E0000010 = Service is in read only mode
E0000010 = Server is unable to respond at the moment.
```
Are both of these used ? Can we remove one of them ?

Reply:
It looks like `errors.E0000010` is for v1 and `E0000010` is OIE-specific (i.e. we can't get rid of one of them).

> Which json mock file has this i18n key ? errors.E0000010

There is no plain API response containing `errors.` prefix. It is being added by v1 [response transformer](https://github.com/okta/okta-signin-widget/blob/master/src/util/Util.js#L80) and this PR introduces it for v2 error transforming code.